### DISCUSSION TITLE
Add Holiday Array

### DISCUSSION THREAD

**Discussion by fredchung1224** [wrick17/calendar-plugin#7] [Ideas]
var defaultConfig = {
...
  holiday: ['12/25/2021', '12/26/2021', '12/27/2021', '1/1/2022'],
...
};

in calendar.js
Line 321 insert the followings
        var holiday = '';
        settings.holiday.forEach(function(item, index) {
          var dayStr = (day.getMonth() + 1) + "/" + day.getDate() + "/" + day.getFullYear()
          if (dayStr == item.toString()) {
            holiday = ' holiday ';
          }
        })

in style.css
Line 27 insert the followings
/* holiday */
.week .holiday {
  color: red !important;
}

**Comment by pratyushok** [comment]
Why does this need to be a part of the library? You can pass custom classes to dates. Add add styles separately.
